spec/helpers/file_helper_spec.rb

```diff
@@ -0,0 +1,38 @@
+describe FileHelper do
+  # Put some confidence in our submission filename sanitization
+  context 'A new file when submitted' do

```

Comment:
Try rephrasing this now so that it's more generic, and doesn't refer to "submitted", since the helper can be applied to all filenames.